packages/xod-client/src/user/actions.js

```diff
@@ -0,0 +1,18 @@
+import url from 'url';
+import { UPDATE_COMPILE_LIMIT } from './actionTypes';
+import { ORIGIN_COMPILE } from './constants';
+
+export const updateCompileLimit = () => (dispatch) => {
+  fetch(url.format({
+    ...url.parse(process
+      ? process.env.ORIGIN_COMPILE || ORIGIN_COMPILE
```

Comment:
Also `ORIGIN_COMPILE` sounds different from `XOD_SITE_DOMAIN`, `XOD_FORUM_DOMAIN`... Logically, it should be something like `XOD_COMPILE_DOMAIN`.

And we've discussed that we should have the only flag just for switching between production/staging hostnames... so it should be something completely different and probably used by new functions in the `siteLinks.js`